File: backend/DataAccess/DaoGenerated/システム共有/tt_aftusinlogDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             通信ログテーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_aftusinlogDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_aftusinlog";
		public long tusinlogseq { get; set; }                     //通信ログシーケンス
		public long sessionseq { get; set; }                      //セッションシーケンス
		public DateTime syoridttmf { get; set; }                  //処理日時（開始）
		public DateTime syoridttmt { get; set; }                  //処理日時（終了）
		public string? msg { get; set; }                          //メッセージ
		public string? request { get; set; }                      //リクエスト
		public string? response { get; set; }                     //レスポンス
		public string? ipadrs { get; set; }                       //IPアドレス
		public string? os { get; set; }                           //OS
		public string? browser { get; set; }                      //ブラウザー情報
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tt_aftusinlogDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             通信ログテーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_aftusinlogDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_aftusinlog"
        Public Property tusinlogseq As Long                     '通信ログシーケンス
        Public Property sessionseq As Long?                      'セッションシーケンス
        Public Property syoridttmf As Date                  '処理日時（開始）
        Public Property syoridttmt As Date                  '処理日時（終了）
        Public Property msg As String                          'メッセージ
        Public Property request As String                      'リクエスト
        Public Property response As String                     'レスポンス
        Public Property ipadrs As String                       'IPアドレス
        Public Property os As String                           'OS
        Public Property browser As String                      'ブラウザー情報
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
    End Class
End Namespace
